src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -34,34 +34,12 @@
 #include <bsoncxx/types/bson_value/view.hpp>
 
 
-namespace {
-
-class Appendable {
-public:
-    virtual ~Appendable() = default;
-    virtual void append(const std::string& key, bsoncxx::builder::basic::document& builder) = 0;
-    virtual void append(bsoncxx::builder::basic::array& builder) = 0;
-};
+namespace genny {
```

Comment:
Namespaces are hard

Reply:
ya..if this has to be in `genny`, consider `genny::value_generators` to keep it a bit more tidy. I'm afraid of `genny::` becoming a `std::` behemoth.